image-load-error-handle

mrliaocn opened the pull request:
1. fixed comment height calculate bug.
2. handle the image loading error by replacing it with default image.

mrliaocn committed:
image-load-error-handle

1. fixed comment height calculate bug.
2. handle the image loading error by replacing it with default image.

cheng-kang reviewed (commented):
SVG file content should be cleaned.

[SVG Minifier](https://jakearchibald.github.io/svgomg/)

cheng-kang committed:
New error image & more avatar onerror handling & naming

1. New error image.
2. Added avatar image onerror handling to `WfReplyArea.vue` and `WfUserSetting.vue`
3. Refined naming

cheng-kang reviewed (changes requested):
Didn't handle avatar onerror in `WfReplyArea.vue` and `WfUserSetting.vue`.
